test/unit/rubygem_searchable_test.rb

```diff
@@ -35,16 +43,10 @@ class RubygemSearchableTest < ActiveSupport::TestCase
         authors:           "Joe User",
         info:              "some description",
         licenses:          "MIT",
-        metadata:          { "foo" => "bar" },
+        metadata:          { "funding_uri" => "http://example.com" },
         sha:               "b5d4045c3f466fa91fe2cc6abe79232a1a57cdf104f7a26e716e0a1e2789df78",
         project_uri:       "http://localhost/gems/example_gem",
         gem_uri:           "http://localhost/gems/example_gem-1.0.1.gem",
-        homepage_uri:      "http://example.com",
-        wiki_uri:          "http://example.com",
-        documentation_uri: "http://example.com",
-        mailing_list_uri:  "http://example.com",
-        source_code_uri:   "http://example.com",
-        bug_tracker_uri:   "http://example.com",
```

Comment:
Now that you have added all metadata uri keys, these should pass. please don't remove it. you will need to update line 46 accordingly.

Reply:
D'oh 🤦